src/view/components/body/td.core.js

```diff
@@ -2,8 +2,9 @@ import Directive from '@grid/view/directives/directive';
 import cellBuilder from '../cell/cell.build';
 import {AppError} from '@grid/core/infrastructure';
 import {VIEW_CORE_NAME, TD_CORE_NAME, GRID_NAME} from '@grid/view/definition';
-import {GRID_PREFIX} from '@grid/core/definition';
-import * as css from '@grid/core/services/css';
+import {TdCtrl} from '@grid/core/cell/td.ctrl';
+ 
+const classify = TdCtrl.classify;
```

Comment:
Somewhere `TdCtrl` methods are called through class reference e.g `TdCtrl.classify(...)`, and somewhere through the variable: `const classify = TdCtrl.classify` .
I think it should be unified

Reply:
yes but here i need performance we can think about this in the future